tardis/io/config_reader.py

```diff
@@ -6,9 +6,9 @@
 
 import tardis
 from tardis.io import config_validator
-from tardis.io.util import YAMLLoader, yaml_load_file
+from tardis.io.util import YAMLLoader, yaml_load_file, quantity_from_str
+from tardis.util import quantity_linspace
 
-pp = pprint.PrettyPrinter(indent=4)
```

Comment:
Don't make changes to files or code that are out of scope for your pull request.